lib/codegen/src/binemit/relaxation.rs

```diff
@@ -95,6 +95,11 @@ pub fn relax_branches(func: &mut Function, isa: &TargetIsa) -> CodegenResult<Cod
         }
     }
 
+    for (jt, jt_data) in func.jump_tables.iter() {
+        func.jt_offsets[jt] = offset;
+        offset += jt_data.len() as u32 * 4; // TODO: size should be based on settings
```

Comment:
Ideally, rather than settings, we'd want the relaxation pass to compute the minimum size that can hold all the offsets, and use that. For now it's sufficient to update the comment :-).